sros2/test/sros2/commands/security/verbs/test_generate_policy.py

```diff
@@ -0,0 +1,79 @@
+# Copyright 2019 Canonical Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import pytest
+import tempfile
+
+import rclpy
+from rclpy.qos import QoSProfile
+from ros2cli import cli
+from std_msgs.msg import String
```

Comment:
We should add a test_depend on std_msgs in package.xml

Reply:
Oh of course, my apologies-- done.